yb-voyager/cmd/analyzeSchema.go

```diff
@@ -259,6 +260,29 @@ func reportSummary() {
 	}
 }
 
+func addSummaryDetailsForIndexes() {
+	var indexesInfo []utils.IndexInfo
+	found, err := metaDB.GetJsonObject(nil, metadb.INDEXES_INFO_KEY, &indexesInfo)
```

Comment:
Suggest renaming this to 
`var sourceIndexesInfo` and `metadb.SOURCE_INDEXES_INFO`

Reply:
`metadb.SOURCE_INDEXES_INFO`? do you mean inplace of `metadb.INDEXES_INFO_KEY`